code/modules/assembly/igniter.dm

```diff
@@ -59,3 +59,6 @@
 	if(!istype(loc, /obj/item/assembly_holder))
 		activate()
 	add_fingerprint(user)
+
+/obj/item/assembly/igniter/get_heat()
+	return 2000
```

Comment:
Can you include this change from 20000 to 2000 in your PR? You might also want to change this back, because this could count as a balance change because of thermite and crap